README.md

```diff
@@ -0,0 +1,27 @@
+
+Introduction
+============
+
+This contains a common parent for all projects to use.
+
+Releasing Projects
+==================
+
+The release profile enables the maven-gpg-plugin, maven-source-plugin and the license-maven-plugin. Note that the license-maven-plugin ( See [here](http://code.mycila.com/license-maven-plugin) ) has a standard format for licenses. An alternative format for the Apache 2 license is available by specifiying
+
+    <plugin>
+        <groupId>com.mycila</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <configuration>
+           <header>APACHE-2-SIMPLIFIED-COPYRIGHT.txt</header>
+        </configuration>
+    </plugin>
+
+in the project configuration. The allows `Copyright (C) ${project.inceptionYear} ${owner}` instead of `Copyright (C) ${project.inceptionYear} ${owner} (${email})`.
+
+Releasing the releng-tools project
+==================================
+
+As the license-plugin must have the license module as a dependency, a project variable is used. This means that when releasing this variable **must** be specified e.g.
+
+    mvn -DdryRun=true release:prepare -DglobalVersion=<next release version>
```

Comment:
We have a module in Indy that is used in a similar way, as a plugin dependency. It specifies a bunch of assembly descriptors that can be reused throughout the build. But, since Indy's parent POM isn't intended to be inherited outside the project, we stuck with project.version.

I'd say if this approach works, use it. I do wonder if you could set a property in the pom like this, though:

```
<properties>
  <globalVersion>${project.version}</globalVersion>
</properties>
```

Would the release plugin pick that up correctly, do you think? If so, it would take that extra part off of the release command line.

If not, then I'd say it would be better if there was a way to use something like an enforcer rule triggered by the release profile to ensure that plugin dep wasn't left as a snapshot version. I can't remember whether we have something like that now.